keras/initializers.py

```diff
@@ -208,7 +208,9 @@ def __call__(self, shape, dtype=None):
         else:
             scale /= max(1., float(fan_in + fan_out) / 2)
         if self.distribution == 'normal':
-            stddev = np.sqrt(scale)
+            # constant taken from scipy.stats.truncnorm.std(a=-2, b=2, loc=0., scale=1.)
+            # this constant is necessary to assure a distribution with the correct standard deviation
+            stddev = np.sqrt(scale) / .87962566103423978
```

Comment:
Inserting this line changes the initial weight values of some models, and it may sometimes be the difference between a model starting to train or not.

Besides that, Keras optimizers should be consistent with TF optimizers (in `tf.init_ops`). So, please revert this line.

Reply:
You are requesting a potentially-breaking behavior change in both Keras and TensorFlow. It's possible, but you will have to get the approval of a number of stakeholders. I suggest you start by opening a well-argued issue on the TensorFlow repo.